inbm/Changelog.md

```diff
@@ -4,14 +4,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## X.X.X - YYYY-MM-DD
-### Changed
-- (NEXMANAGE-956) Improve failure reason from TC that is logged in Mjunct DB for FOTA
-- (NEXMANAGE-1194) provision-tc/mqtt services will now handle UDM 
+### Added
+ - (NEXMANAGE-1222) Add support for Ubuntu 24.04
 
 ## 4.2.8.2 - 2024-12-18
 ### Changed
- - (NEXMANAGE-1121) Remove mqtt-ca group from /var/cache/manageability/repository-tool directory
- - Update CI setup script to install rchardet v1.8.0
+- (NEXMANAGE-956) Improve failure reason from TC that is logged in Mjunct DB for FOTA
+- (NEXMANAGE-1194) provision-tc/mqtt services will now handle UDM 
```

Comment:
move NEXMANAGE-956/NEXMANAGE-1194 back to the 'next' release (they are not in 4.2.8.2)